src/hooks/useENSPendingRegistrations.tsx

```diff
@@ -0,0 +1,33 @@
+import { useSelector } from 'react-redux';
+import { useAccountSettings } from '.';
+import { ENSRegistrationState } from '@rainbow-me/entities';
+import { AppState } from '@rainbow-me/redux/store';
+
+export default function useENSPendingRegistrations() {
```

Comment:
this one gives us the registrations that started (as commit transaction sent) but is not finished yet (as registerWithConfig transaction not sent yet)